### DISCUSSION TITLE
Mathematical expressions - text decorations stopped working

### DISCUSSION THREAD

**Discussion by cxdorn** [community/community#136298] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

A week ago, writing `$\textit{a} = \texttt{a} = \textbf{a}$` in markdown would preview the letter `a` in different text decorations (mirroring the corresponding Latex commands).  
Now it looks like this: $`\textit{a} = \texttt{a} = \textbf{a}`$, i.e. all the decoration look the same (like `\textrm{a}`). What changed? How can I get back the old behaviour?

**Comment by CopperEagle** [comment]
Hey @cxdorn 

Looks quite distinct on my end... (see screenshot below). Is that what you mean? What setup are you using?

![Screenshot from 2024-08-19 16-23-24](https://github.com/user-attachments/assets/8ec893c7-193d-42a0-bbe8-a9c1b642b667)

**Comment by cxdorn** [comment]
Thank you @CopperEagle ! I'm getting this behaviour on Chrome 127 (also on a fresh install/without any extensions). On Firefox though everything is displayed correctly. Chrome screenshot below:
<img width="300" alt="Screenshot 2024-08-20 at 10 05 43" src="https://github.com/user-attachments/assets/bf931133-93aa-4f35-bf0c-2f1b3b5e2f20">

In the developer tools, no errors are reported. And I find it surprising that the font decoration _is_ changed (namely to `\textrm`).
